test/e2e/limit_range.go

```diff
@@ -93,17 +93,122 @@ var _ = framework.KubeDescribe("LimitRange", func() {
 
 		By("Failing to create a Pod with less than min resources")
 		pod = f.NewTestPod(podName, getResourceList("10m", "50Mi"), v1.ResourceList{})
-		pod, err = f.ClientSet.Core().Pods(f.Namespace.Name).Create(pod)
+		pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(pod)
 		Expect(err).To(HaveOccurred())
 
 		By("Failing to create a Pod with more than max resources")
 		pod = f.NewTestPod(podName, getResourceList("600m", "600Mi"), v1.ResourceList{})
-		pod, err = f.ClientSet.Core().Pods(f.Namespace.Name).Create(pod)
+		pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(pod)
 		Expect(err).To(HaveOccurred())
 	})
 
 })
 
+var _ = framework.KubeDescribe("LimitRange [Feature:LocalStorageCapacityIsolation]", func() {
+	f := framework.NewDefaultFramework("limitrange")
+
+	BeforeEach(func() {
+		// only run the tests when LocalStorageCapacityIsolation feature is enabled
+		framework.SkipUnlessLocalEphemeralStorageEnabled()
+	})
+
+	It("should create a LimitRange with default ephemeral storage and ensure pod has the default applied.", func() {
```

Comment:
A large amount of the code below is duplicated from the It block above. It seems reasonable to either use a

Reply:
For now, since the ephemeral storage feature is alpha, i add a new  `KubeDescribe` block, and when the feature graduates to beta or GA, i will merge them, should we do that now ?

Reply:
I guess it is not needed now unless you think there is a chance that after GA you won't have time to consolidate the code. It is not critical but it usually preferred to extract out common patterns (DRY).